### PR TITLE
Re #176; Update AI2 name in footer.

### PR DESCRIPTION
**Pull request by codeviking**
Use the canonical form of AI2's name in the
`<Footer />`.

### CODE REVIEW

**Comment by codeviking** [comment]
We should likely wait to merge this until legal approves it, @aaronsarnat has reached out.

**Comment by aaronsarnat** [comment]
Yeah, as @codeviking mentioned, it's possible we can't do this from a legal perspective. Either way, we probably want to get rid of the `(AI2)` text.

I will reach out to Jen again today to hopefully get final word on org name policy in copyright line.

File: components/Footer.tsx [hunk 1]
```diff
@@ -52,9 +52,9 @@ export class Footer extends React.PureComponent<Props> {
                             this.props.children
                         ) : (
                             <Body>
-                                © The{' '}
+                                ©{' '}
                                 <ExternalLink contrast={contrast} href="https://allenai.org">
-                                    Allen Institute for Artificial Intelligence (AI2)
+                                    Allen Institute for AI
```
Comment (aaronsarnat) [inline components/Footer.tsx]: Got final word from Jen. This line is considered legal copy, and as such, needs to reference our legal entity name, not our friendly DBA name. So this reference should be:

```
The Allen Institute for Artificial Intelligence
```

Which includes "The" at the beginning, does not use the "AI" acronym, and does not include "(AI2)."

Comment (codeviking) [reply components/Footer.tsx]: 🤘 Updated, please confirm.

**Comment by codeviking** [comment]
@aaronsarnat Take another look, and let me know if it's good to go!

**Pull request merged**